scripts/bygg

```diff
@@ -12,17 +12,50 @@ SCRIPT_FOLDER=$(dirname "$SCRIPTNAME")
 PLATFORM_FOLDER=$(realpath "$(dirname "$SCRIPT_FOLDER")")
 
 # Parse options or show usage which exits app
-usage()
-{
-  echo "$0 - build bitbake recipes/images using docker"
-  echo "Usage: $(basename "$0" ) [options] build-command-to-run"
-  echo "options(: means required argument, :: means optional argument)"
-  for option in $(echo "$LONGOPTIONS" | tr , \\n); do
-    echo "--$option"
-  done
+usage() {
+
+  cat << EOF
+$0 - build bitbake recipes/images using docker
+Usage: $(basename "$0") [options] build-command-to-run
+options(: means required argument, :: means optional argument)
+
+OPTIONS:
+  -b, --build-folder     Optional, specify the folder for the build output. Default is 'build' directory.
+  -c, --delete-conf      Optional, delete existing configuration files in build-folder. Recomended to use if built diffrent machines and multiple machines.
```

Comment:
Delete existing configuration files in the build directory. Recommended if you build multiple different machines.